pkg/common/funcs.go

```diff
@@ -28,37 +27,34 @@ func GetFalconImage(nodesensor *falconv1alpha1.FalconNodeSensor) string {
 	return nodesensor.Spec.Node.Image
 }
 
-func FalconSensorConfig(falconsensor *falconv1alpha1.FalconSensor) (map[string]string, error) {
-	m := make(map[string]string)
-	var cmOptInt map[string]interface{}
-	jsonCmOpt, err := json.Marshal(falconsensor)
-	if err != nil {
-		return nil, err
+func FalconSensorConfig(falconsensor *falconv1alpha1.FalconSensor) map[string]string {
```

Comment:
@isimluk what do you think about this function?

Reply:
👍